tensorrt_llm/_torch/modules/encoder_decoder_layer.py

```
@@ -0,0 +1,55 @@
+
+import torch
+from torch import nn
+
+from ..attention_backend import AttentionMetadata
+
+
+class EncoderLayer(nn.Module, ABC):
+    """Abstract base class for encoder layers (self-attention only, non-causal)."""
+
+    @abstractmethod
+    def forward(
```

Flake8 (7.3.0)

[error] multiple statements on one line (def)

(E704)